src/modules/ethereum/loanTransactions.js

```diff
@@ -88,7 +88,7 @@ function parseProducts(productsArray) {
                 discountRate,
                 bn_collateralRatio,
                 collateralRatio: bn_collateralRatio / PPM_DIV,
-                minDisbursedAmountInToken: bn_minDisbursedAmount / DECIMALS_DIV,
+                minDisbursedAmountInToken: bn_minDisbursedAmount / DECIMALS_DIV * 1.1,
```

Comment:
for readability maybe add it to`utils/constants.js`  as `MIN_LOAN_AMOUNT_ADJUSTMENT` and comment the rational: it's to avoid loan tx to fail on min loan amount because of an ETH/EUR rate change in the background right when sending the tx from the UI

also, apologies but I did a bit of math and 25% is the ideal number, so make it 1.25 :)